src/main/java/com/github/dockerjava/api/model/Container.java

```diff
@@ -30,11 +34,17 @@
     @JsonProperty("Image")
     private String image;
 
+    /**
+     * @since since {@link RemoteApiVersion#VERSION_1_21}
+     */
+    @JsonProperty("ImageID")
+    private String imageId;
+
     @JsonProperty("Names")
     private String[] names;
 
     @JsonProperty("Ports")
-    public Port[] ports;
+    public ContainerPort[] ports;
```

Comment:
I see no reasons for it. I thought about subpackage, but then decided just to prefix it. I used reflection methods for equals/hashcode/toString (hope they wouldn't used often and didn't affect performance or security) but if we would need switch back to *Builders, then will add hundreds of lines. As i understand nested class can be only final or static for successful deserialisation?
Container probably is the most huge class so i slitted it, small classes i kept nested.
PS HostConfig seems reused in request as i saw somewhere not existed fields. So it probably would require to split it and subclass will be useful then. When i'm reviewing fields i go into cycles and start thinking that 2 monitors isn't enough for all API pages :) 
PPS I also found this https://github.com/docker/engine-api/tree/master/types and will try compare with what we have.